android-components/components/browser/state/src/main/java/mozilla/components/browser/state/engine/middleware/TranslationsMiddleware.kt

```diff
@@ -365,6 +380,57 @@ class TranslationsMiddleware(
         }
     }
 
+    /**
+     * Retrieves the download size and dispatches the result to the
+     * store via [TranslationsAction.SetTranslationDownloadSizeAction].
```

Comment:
```suggestion
     *[BrowserStore] via [TranslationsAction.SetTranslationDownloadSizeAction].
```
I think we should be clear which store we are talking about given all the store we have.

Reply:
Updated, agreed, extremely important now!